### PR TITLE
Fix cluster claim issue

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
@SanjalKatiyar @bipuladh Please review

File: packages/mco/components/data-policies/create-dr-policy/select-replication-type.tsx [hunk 1]
```diff
@@ -186,7 +186,7 @@ export const DRReplicationType: React.FC<DRReplicationTypeProps> = ({
                   { name: c?.name, version: ODF_MINIMUM_SUPPORT }
                 )
               )
-            : !c?.cephFSID &&
+            : c?.cephFSID === '' &&
```
Comment (SanjalKatiyar) [inline packages/mco/components/data-policies/create-dr-policy/select-replication-type.tsx:189]:
```suggestion
            : c.cephFSID === '' &&
```
`?.` not required... but it is an optional change...

**Pull request merged**